src/Kernel/Behavior.class.st

```diff
@@ -631,6 +631,12 @@ Behavior >> deepCopy [
 	^ self shallowCopy
 ]
 
+{ #category : #'accessing instances and variables' }
+Behavior >> definedVariables [
+	"return all the Variables defined"
+	^ self slots
```

Comment:
It should use #localSlots. Class does not define variables of its traits.
For all variables we need another method #ownedVariables (with name following slot property #owningClass).

Reply:
Thanks! added to backlog